paramak/utils.py

```diff
@@ -187,32 +187,34 @@ def extend(point_a: Tuple[float, float], point_b: Tuple[float, float],
 
 def find_center_point_of_circle(point_a: Tuple[float, float],
                                 point_b: Tuple[float, float],
-                                point3: Tuple[float, float]) -> Tuple[Tuple[float, float], float]:
+                                point_c: Tuple[float, float]) -> Tuple[Tuple[float, float], float]:
     """
     Calculates the center and the radius of a circle
     passing through 3 points.
     Args:
         point_a (float, float): point 1 coordinates
         point_b (float, float): point 2 coordinates
-        point3 (float, float): point 3 coordinates
+        point_c (float, float): point 3 coordinates
     Returns:
         (float, float), float: center of the circle coordinates or
         None if 3 points on a line are input and the radius
     """
 
     temp = point_b[0] * point_b[0] + point_b[1] * point_b[1]
     bc = (point_a[0] * point_a[0] + point_a[1] * point_a[1] - temp) / 2
-    cd = (temp - point3[0] * point3[0] - point3[1] * point3[1]) / 2
-    det = (point_a[0] - point_b[0]) * (point_b[1] - point3[1]) - (
-        point_b[0] - point3[0]
+    cd = (temp - point_c[0] * point_c[0] - point_c[1] * point_c[1]) / 2
+    det = (point_a[0] - point_b[0]) * (point_b[1] - point_c[1]) - (
+        point_b[0] - point_c[0]
     ) * (point_a[1] - point_b[1])
 
     if abs(det) < 1.0e-6:
         return (None, np.inf)
 
     # Center of circle
-    cx = (bc * (point_b[1] - point3[1]) - cd * (point_a[1] - point_b[1])) / det
-    cy = ((point_a[0] - point_b[0]) * cd - (point_b[0] - point3[0]) * bc) / det
+    cx = (bc * (point_b[1] - point_c[1]) -
+          cd * (point_a[1] - point_b[1])) / det
+    cy = ((point_a[0] - point_b[0]) * cd -
+          (point_b[0] - point_c[0]) * bc) / det
```

Comment:
Should this be in a seperate PR or is it fine  here?

Reply:
You caught me trying to sneak in what should be a separate PR. I can remove it and add it as a separate PR